test/fixtures/modifyVars.less

```diff
@@ -1,8 +1,9 @@
 @imgPath: '/path/to/images';
 @customPadding: 12px;
+@customPadding: undefined;
```

Comment:
Can't you add a new test target and set the variable there to `undefined`? This is certainly not enough.

Also, when you are done, please fetch upstream, rebase and squash.